export const DomainException = {
  USER_ALREADY_EXISTS: { code: "10", message: "Usuário já existe" },
};
